Core/MIPS/JitCommon/JitCommon.cpp

```diff
@@ -232,4 +232,14 @@ std::vector<std::string> DisassembleX86(const u8 *data, int size) {
 	return lines;
 }
 
+#elif defined(MIPS)
+
+std::vector<std::string> DisassembleMIPS(const u8 *data, int size) {
+	std::vector<std::string> lines;
+	// TODO: A MIPS disassembler for MIPS32r2+
+	// Utilise MIPSDisAsm(MIPSOpcode op, u32 pc, char *out, bool tabsToSpaces)
```

Comment:
Personally, I think a disassembler that's missing a few things is a lot better than none at all.  But, if we want it to be accurate, maybe we handle them like this:

```c++
char temp[512];
if (!MIPS64OverrideDisAsm(op, pc, temp)) {
    // Okay, this isn't different from the PSP, so just use the same disasm.
    MIPSDisAsm(op, pc, temp);
}
lines.push_back(temp);
```

Where we would slowly build out `MIPS64OverrideDisAsm`.  Its job would be:
1. Output invalid for opcodes that MIPSDisAsm would think are vfpu/etc.
2. Output valid opcodes for things the PSP lacks.

Everything else would just be sent to the PSP func, which would work totally fine.

-[Unknown]